bin/example.rb

```diff
@@ -42,36 +43,36 @@ def send_message
 #
 def send_multiple_messages
 
-    puts "EXECUTING SEND MULTIPLE MESSAGES..."
-
-    client = Messagemedia::SOAP::Client.new(USER_ID, PASSWORD)
-
-    # Construct the message
-    message1 = Messagemedia::SOAP::Message.new
-    message1.content = "Content of Message 1 to Recipient 1"
-    message1.delivery_report = true
-    message1.format = Messagemedia::SOAP::FORMAT_SMS
-    message1.validity_period = 1
-    # (Optional) This attribute specifies a sequence number that is assigned to the message and is used to identify the message if an error occurs. Each message error in the response will specify the sequence number of the message that caused the error. Sequence numbers should be unique within the request. 1 to 2147483647.
-    message1.sequence_number = 1
-    # (Optional) This element specifies the message source address. The specified address will be used wherever possible, however due to limitations with various carriers, legislation etc, the final message is not guaranteed to come from the specified address.
-    message1.origin = "Origin_1"
-    message1.add_recipient(1234567890, DESTINATION_NUMBER)
-
-    # Construct the message
-    message2 = Messagemedia::SOAP::Message.new
-    message2.content = "Content of Message 2 to Recipient 2"
-    message2.delivery_report = true
-    message2.format = Messagemedia::SOAP::FORMAT_SMS
-    message2.validity_period = 1
-    # (Optional) This attribute specifies a sequence number that is assigned to the message and is used to identify the message if an error occurs. Each message error in the response will specify the sequence number of the message that caused the error. Sequence numbers should be unique within the request. 1 to 2147483647.
-    message2.sequence_number = 1
-    # (Optional) This element specifies the message source address. The specified address will be used wherever possible, however due to limitations with various carriers, legislation etc, the final message is not guaranteed to come from the specified address.
-    message2.origin = "Origin_2"
-    message2.add_recipient(234567890, DESTINATION_NUMBER)
-
-    result = client.send_messages([ message1, message2 ])
-    display_result result
+  puts "Executing send multiple messages..."
+
+  client = Messagemedia::SOAP::Client.new(USER_ID, PASSWORD)
+
+  # Construct the message
+  message1 = Messagemedia::SOAP::Message.new
+  message1.content = "Content of Message 1 to Recipient 1"
+  message1.delivery_report = true
+  message1.format = Messagemedia::SOAP::FORMAT_SMS
+  message1.validity_period = 1
```

Comment:
I know this was what we had before, but what time unit is this? Would it be better to set the demo to a more realistic value?